tests/integration_tests/tests/agentless_tests/component/test_component_basics.py

```diff
@@ -84,8 +87,8 @@ def test_component_creation_with_secrets_and_plugins(self):
         self.assertTrue(self.client.deployments.get(self.component_name))
         self.assertEqual(self.client.secrets.get('secret1')['value'], 'test')
         plugins_list = self.client.plugins.list()
-        self.assertEqual(len(plugins_list), 1)
-        self.assertTrue(plugins_list[0]['package_name'],
+        self.assertEqual(len(plugins_list), 2)
+        self.assertTrue(plugins_list[1]['package_name'],
```

Comment:
we can't assert on the plugin list like that because it used to be `1`, now it's `2`, but if you also have another test in this run which uploads another plugin before, then it'll be `3` or `4` or...

instead, do something like `assert any(p['package_name'] == 'cloudify-openstack-plugin' for p in plugins_list)`, or add a package_name filter to that restclient `.list()` call

Reply:
Done and updated